src/components/SpruceForm/FieldTemplates.tsx

```diff
@@ -154,7 +154,7 @@ const ArrayContainer = styled.div`
 const DeleteButtonWrapper = styled(ElementWrapper)`
   margin-left: ${size.s};
   // Align button with top of input unless it should specifically align to the top of the ArrayItemRow
-  align-items: ${({ topAlignDelete }: { topAlignDelete: boolean }) =>
+  margin-top: ${({ topAlignDelete }: { topAlignDelete: boolean }) =>
     topAlignDelete ? "0px" : "20px"};
```

Comment:
This is a common pattern we have been doing with lg components  i wonder if we could make it more universal. (Not actionable just a thought i had when i saw this)